march news (#1273)
ropensci/roweb3: content/blog/2026-03-30-news-mars-2026/index.Rmd

    stringr::str_pad(lubridate::day(rmarkdown::yaml_front_matter(knitr::current_input())$date), 2, "0", side = "left"),
    rmarkdown::yaml_front_matter(knitr::current_input())$slug
    )
english <- function(x) {
  as.character(english::english(x))
}

nice_string <- function(...) {
  if (length(...) == 2) {
    return(paste(c(...), collapse = " and "))
  }
  glue::glue_collapse(trimws(...), sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### rOpenSci Dev Guide 1.0.0: Trilingual and Improved

rOpenSci Software Peer Review’s guidance is gathered in an [online book](https://devguide.ropensci.org/) that keeps improving! 
It is now available in English, [Spanish](https://devguide.ropensci.org/es/index.es.html) and [Portuguese](https://devguide.ropensci.org/pt/index.pt.html).
Read more in the [release announcement](/blog/2026/03/02/devguide-1.0.0/)

### Champions Program Update

> TODO YANI

Reviewer: ```suggestion
We are still going through the Champions selection process, and we’re excited to share that the new group of mentors has already been selected and is now actively reviewing Champions applications.

This cohort brings together a wonderful mix of returning Champions stepping into mentorship roles, mentors continuing their contributions, and new members joining the program. The 2026 mentors are Andrea Gómez Vargas, Pablo Paccioretti, Alber Hamersson Sánchez Ipia, Erick Isaac Navarro Delgado, Francisco Cardozo, Luis Verde Arregoitia, Monika Ávila Márquez, Guadalupe Pascal, Pao Corrales, and Elio Campitelli. Together, they represent a diverse and vibrant community across Colombia, Mexico, Argentina, Brazil, and Bolivia, with some currently based in Switzerland, Canada, the United States, and Australia. We’re very happy to see this growing, interconnected network supporting the next cohort of Champions.
```

knitr::opts_chunk$set(echo = FALSE)
url <- sprintf(
    "/blog/%s/%s/%s/%s",
    lubridate::year(rmarkdown::yaml_front_matter(knitr::current_input())$date),
    stringr::str_pad(lubridate::month(rmarkdown::yaml_front_matter(knitr::current_input())$date), 2, "0", side = "left"),
    stringr::str_pad(lubridate::day(rmarkdown::yaml_front_matter(knitr::current_input())$date), 2, "0", side = "left"),
    rmarkdown::yaml_front_matter(knitr::current_input())$slug
    )
english <- function(x) {
  as.character(english::english(x))
}

nice_string <- function(...) {
  if (length(...) == 2) {
    return(paste(c(...), collapse = " and "))
  }
  glue::glue_collapse(trimws(...), sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### rOpenSci Dev Guide 1.0.0: Trilingual and Improved

rOpenSci Software Peer Review’s guidance is gathered in an [online book](https://devguide.ropensci.org/) that keeps improving! 

> It is now available in English, [Spanish](https://devguide.ropensci.org/es/index.es.html) and [Portuguese](https://devguide.ropensci.org/pt/index.pt.html).

Reviewer 2: ```suggestion
rOpenSci Software Peer Review’s guidance is gathered in an online book that keeps improving! 
It is now available in [English](https://devguide.ropensci.org/), [Spanish](https://devguide.ropensci.org/es/index.es.html) and [Portuguese](https://devguide.ropensci.org/pt/index.pt.html).
```

Read more in the [release announcement](/blog/2026/03/02/devguide-1.0.0/)

### Champions Program Update

We are still going through the Champions selection process, and we’re excited to share that the new group of mentors has already been selected and is now actively reviewing Champions applications.

This cohort brings together a wonderful mix of returning Champions stepping into mentorship roles, mentors continuing their contributions, and new members joining the program. The 2026 mentors are Andrea Gómez Vargas, Pablo Paccioretti, Alber Hamersson Sánchez Ipia, Erick Isaac Navarro Delgado, Francisco Cardozo, Luis Verde Arregoitia, Monika Ávila Márquez, Guadalupe Pascal, Pao Corrales, and Elio Campitelli. Together, they represent a diverse and vibrant community across Colombia, Mexico, Argentina, Brazil, and Bolivia, with some currently based in Switzerland, Canada, the United States, and Australia. We’re very happy to see this growing, interconnected network supporting the next cohort of Champions.

> ### R-universe update

Reviewer 2: ```suggestion
### R-Universe update
```

> You can now download artifacts and log files from R-universe without being logged in with a GitHub account, for example https://ropensci.r-universe.dev/opencv#checktable.

Reviewer 2: ```suggestion
You can now download artifacts and log files from R-Universe without being logged in with a GitHub account, for example https://ropensci.r-universe.dev/opencv#checktable.
```

All updates are intended to permit generative AI tools to be used in any useful way, while minimising the burden on those who volunteer their own time to keep our software peer review service running. 

### Software review bot updates

The `ropensci-review-bot` now delivers an initial report to all new software pre-submissions and submissions, identifying the five most similar packages from both all rOpenSci packages, and all CRAN packages.
The matches are generated by our [ropensci-review-tools/pkgmatch package](https://docs.ropensci.org/pkgmatch) (itself reviewed in [this review issue](https://github.com/ropensci/software-review/issues/671)).
Matching is based on an ["term frequency-inverse document frequency" algorithm](https://en.wikipedia.org/wiki/Tf%E2%80%93idf), using inverse document frequencies from all rOpenSci and CRAN packages.
Similar package reports can also be manually triggered (by editors only) with `@ropensci-review-bot similar packages`, like in [this example for the pkgmatch package itself](https://github.com/ropensci/software-review/issues/671#issuecomment-4117805740).

### Coworking 

Read [all about coworking](/blog/2023/06/21/coworking/)!

> - Tuesday April 7th 2026, 9:00 Americas Pacific (16:00 UTC) ["Getting to know the CSID Network"](/events/coworking-2026-04/) with [Steffi LaZerte](/author/steffi-lazerte/) and cohost [Irene Ramos](/author/irene-ramos/).  

Reviewer 2: ```suggestion
- Tuesday April 7th 2026, 9:00 Americas Pacific (16:00 UTC) ["Getting to know the CSID Network"](/events/coworking-2026-04/) with [Steffi LaZerte](/author/steffi-lazerte/) and cohosts [Irene Ramos](/author/irene-ramos/) and [Adamu Saleh Saidu](/author/adamu-saleh-saidu).  
```

    -   Learn more about the [CSID Network](https://csidnet.org/)

>     -   Meet cohost, Irene Ramos, and learn more about the CSID Network and how you might get involved.

Reviewer 2: ```suggestion
    -   Meet cohosts, Irene Ramos and Adamu Saleh Saidu, and learn more about the CSID Network and how you might get involved.
```
